honeybee_3dm/togeometry.py

```diff
@@ -1,34 +1,93 @@
-"""Functions to create Ladybug geometries from Rhino geometries."""
+"""Functions to create Ladybug geometries from Rhino3dm geometries."""
 
-from ladybug_geometry.geometry3d.pointvector import Vector3D, Point3D
-from ladybug_geometry.geometry3d.line import LineSegment3D
-from ladybug_geometry.geometry3d.mesh import Mesh3D
-from ladybug_geometry.geometry3d.face import Face3D
-from ladybug_geometry.geometry3d.polyface import Polyface3D
+# The Rhino3dm library provides the ability to access content of a Rhino3dm
+# file from outside of Rhino
+import rhino3dm
 
-import ladybug.color as lbc
+# Importing Ladybug geometry libraries
+# The Ladybug geometry objects are foundation to other objects in Ladybug and Honeybee
+from ladybug_geometry.geometry3d.pointvector import Point3D, Vector3D
+from ladybug_geometry.geometry3d.face import Face3D
 
 
 def to_point3d(point):
-    """Ladybug Point3D from Rhino Point3d."""
-    pass
+    """This function creates a Ladybug Point3D object from a rhino3dm point.
+
+    Args:
+        point (point3d): A rhino3dm point
+
+    Returns:
+        Point3D: A Ladybug Point3D object
+    """
+    return Point3D(point.X, point.Y, point.Z)
+
+
+def to_vector3d(vector):
+    """This function creates a Ladybug Vector3D from a rhino3dm Vector3d.
+
+    Args:
+        vector (vector3d): A rhino3dm vector3d
+
+    Returns:
+         Vector3D: A Ladybug Vector3D object
+    """
+    return Vector3D(vector.X, vector.Y, vector.Z)
+
+
+def mesh_to_face3d(mesh):
+    """This function creates a Ladybug Face3D object out of a rhino3dm Mesh.
+
+    Args:
+        mesh (mesh): A rhino3dm mesh geometry
+
+    Returns:
+        A list: A list of Ladybug Face3D objects
+    """
+
+    faces = []
+    pts = [to_point3d(mesh.Vertices[i]) for i in range(len(mesh.Vertices))]
+
+    for j in range(len(mesh.Faces)):
+        face = mesh.Faces[j]
+        if len(face) == 4:
+            all_verts = (pts[face[0]], pts[face[1]],
+                         pts[face[2]], pts[face[3]])
+        else:
+            all_verts = (pts[face[0]], pts[face[1]], pts[face[2]])
+
+        faces.append(Face3D(all_verts))
+
+    return faces
```

Comment:
Should we return a Ladybug Mesh instead of a list of faces?

Reply:
Honeybee [Face](https://www.ladybug.tools/honeybee-core/docs/honeybee.face.html) module takes a Ladybug Face3D object.